src/components/footer/_footer.scss

```diff
@@ -17,15 +17,15 @@ $include-html: false !default;
         margin-left: auto;
         margin-right: auto;
 
-        padding: 40px $layoutDefaultPadding;
+        padding: rhythm(1) $layoutDefaultPadding;
       }
     }
 
     &__line {
-      margin-bottom: 10px;
+      margin-bottom: rhythm(1);
```

Comment:
This sizes may be tweaked as soon as footer will be updated in projects to see the actual layout.

VR here is not so important, but done for consistency